communications/newsletters/newsletter_12_Nov2019.md

```diff
@@ -0,0 +1,100 @@
+In addition to its previous focus on reproducibility, this project will aim to capture the vast amounts of knowledge of data science practices in the areas of project design, communication, outreach and ethics.
+Following an open source, community driven approach, we will welcome contributions from anyone from within the Turing community and beyond  who can support researchers and practitioners in learning or improving the "how" of conducting world-leading research in data science.
+The project proposal with further details is available [online](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/tps-funding-application-20190429.md).
+
+To plan the next steps, the core team met last month on 11 and 12 October to brainstorm and discuss the next steps in this project.
+The participants of this meeting explored a number of topics in four themed sessions, draft report from which is available on [this HackMD](https://hackmd.io/zVTeKhG2SIiBLam1YtILNg).
+
+![]()
+
+## Relevant resources
+
+### Open Life Science Mentoring Program | Deadline Dec 8
+
+We are delighted to collaborate with the [Open Life Science Program](https://openlifesci.org) organised by Bérénice Batut, Yo Yehudi and Malvika Sharan as one of the [Mozilla Open Leader X projects](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/).
+They are developing this mentoring program for mentoring and training early-stage researchers and potential academic leaders who want to become ambassadors for Life Science in their communities.
+The applications can be submitted via [EasyChair](https://easychair.org/cfp/ols-2020) until **December 8**.
+Reach out to the core team by emailing [hello@openlifesci.org](mailto:hello@openlifesci.org) or connect with them on Twitter [@openlifesci](https://twitter.com/openlifesci).
+
+## Tips & Tricks for new members
+
+### How to best cite Turing Way?
+
+Like our community members, Paola ([GitHub pull request](https://github.com/alan-turing-institute/the-turing-way/issues/648)) and Heidi ([Tweet](https://twitter.com/HeidiBaya/status/1189116056641658880)), have you also wondered how to cite Turing way?
+It is a fair question to ask when referring to an online Open Source project that undergoes several updates in the same week or sometimes on the same day. 
+
+All versions of this book (available under the [cc-by license]((http://creativecommons.org/licenses/by/4.0/legalcode))) can be cited using the DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
+This DOI represents all versions, and will always resolve to the latest one.
+Different versions can be found on [Zenodo](https://zenodo.org) and can be individually cited.
+
+(Kirstie's very sorry that she's behind with releasing some of the versions 😩!
+If you attended a book sprint or have written chapters for the book your name will appear in that author list soon.
+Promise!)
```

Comment:
I added this - is this ok?

Reply:
Looks good. And Kirstie is doing the best she can so I will remove the part where you don't need to be sorry at all <3